Updates in the configuration:

Javilete opened the pull request:
- In the new platform there is only one stub smtp service
- Use only one config file (it is necessary to ssh to the right box on 8888 as it is not exposed through the proxy)
- Update README file with guidelines
- Remove from email service one method as now it is only one service

javierlar committed:
Updates in the configuration:
- In the new platform there is only one stub smtp service
- Use only one config file (it is necessary to ssh to the right box on 8888 as it is not exposed through the proxy)
- Update README file with guidelines
- Remove from email service one method as now it is only one service

mszygenda commented on services/emailService.js:
```diff
@@ -1,34 +1,29 @@
-        });
-    };
-
     var getIdamLatestEmail = function (userEmail, subject) {
```
I think we should keep both functions. The reason for that is our (IDAM emails) are not HTML and because of that the JSON produced by ISG Stub is wrong (newlines in JSON are not escaped) and it fails to parse it otherwise. That's why replace newlines with BR's. IDP Emails on the other hand are already HTML so no further processing should be done.

eg. our email:

```
Hello John,

How are you?

This is unmanned email
```

JSON: { message: "
Hello John,

How are you?

This is unmanned email
"} <-- Real newlines breaks parser
IDP Email:

```
Hello John,<br />How are you?<br />This is unmanned email
```

JSON:

```
{ message: "Hello John,<br />How are you?<br />This is unmanned email" }
```

Javilete replied:
As now we have only one stub email service handling both type of emails (form idp and our events) I have moved everything to the same method, so basically the replace to resolve the double dot and new lines are done in the same method.

mszygenda replied:
ok, if works fine both kinds (no unwanted br's added) then it's fine. i'll merge.

mszygenda commented on common/httpClient.js:
```diff
@@ -14,7 +14,6 @@ module.exports = {
         request(
             extend({
                 rejectUnauthorized: false,
-                proxy: config.httpClient.proxy,
```
I'd leave the possibility to set the proxy (if null or absent in config it wont make a difference)

Javilete replied:
I will set it to null so in the future if it is required it can be easyly sportted where to update it

mszygenda commented on README.md:
```diff
@@ -96,21 +96,21 @@ If you want to change the username simply refresh szygiMail and you'll be again
-    config/int.js - for integration environment
-    config/pr1.js - for pr1 environment
+    config/nft.js - for integration environment
+    config/nftpr1.js - for pr1 environment
```
we no longer have that, right?

Javilete replied:
True, removed

javierlar committed:
Updates:
- Include proxy (even if it is null)
- Improve emailService to do the replace of double dots and new lines of response messages
- README updates regarding config
